src/controllers/remove.director.controller.ts

```diff
@@ -85,7 +85,6 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
 
     // Patch filing with etag and resignation date
     const updateFiling: OfficerFiling = {
-      referenceEtag: appointment.etag,
       resignedOn: resignationDate
```

Comment:
I thought the etag was needed for the termination, but if not.. ought the comment above this be amended?

Reply:
You're right it is still needed. The etag has been moved into the original POST when the 'remove director' button is first clicked to match the new CH01 post. But you're right I'll update the comment.